### ISSUE TITLE
Performance issue if header `Content-Type: application/json` was set

### ISSUE THREAD

**Issue by JanuLakshmanan** [xmendez/wfuzz#238]
## Performance issue if header `Content-Type: application/json` was set

**Please check:**

- [x] I've read the docs for [Wfuzz](http://wfuzz.readthedocs.io/)

**Please describe your local environment:**

Wfuzz version: 2.4.7

Python version: 3.8.5

OS: windows 10

## Report

**What is the current behavior?**

 wfuzz -z file,wordlist/others/common_pass.txt -u <url> -H "Content-Type: Application/json" -X POST -d '{"userName":"admin","password":"FUZZ"}'

It is not getting response code 200 even for a valid payload.

**What is the expected or desired behavior?**

Need to get 200 response

**Please provide steps to reproduce, including exact wfuzz command executed and output:**

 wfuzz -z file,wordlist/others/common_pass.txt -u <url> -H "Content-Type: Application/json" -X POST -d '{"userName":"admin","password":"FUZZ"}'

Only i am getting 500 as response code for all my payloads

**Comment by xmendez** [comment]
There were bugs in old version of wfuzz when dealing with json payload. Can you please update wfuzz?
cheers

**Comment by JanuLakshmanan** [comment]
> There were bugs in old version of wfuzz when dealing with json payloads. Do you get the same with newer versions?
> Seems duplicated by #227
> 
> cheers

Yes, I am using the new version of Wfuzz. Still facing the issue.
